src/sentry/web/frontend/project_settings.py

```diff
@@ -186,9 +205,12 @@ def handle(self, request, organization, team, project):
                     'scrub_data',
                     'scrub_defaults',
                     'sensitive_fields',
-                    'scrub_ip_addresses',
+                    'scrub_ip_address',
                     'scrape_javascript',
                     'blacklisted_ips'):
+                # Value can't be overridden if set on the org level
+                if opt in form.org_overrides and organization.get_option('sentry:%s' % (opt,), False):
```

Comment:
we should add a test for submitting a value here when the override is set, just to make sure it doesnt apply